CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ This changelog format is largely based on [Keep A Changelog](https://keepachange
 
 #### 💥 Breaking Changes
 
+- No longer defaults package manager to pnpm if no flag is passed
```

Comment:
I think we should continue to have a default here, if no config or flag is passed

Reply:
I'm hesitant to assume a project's package manager. If the script defaults to pnpm when the project is actually using yarn the script will still error, but it may be less obvious than an unsupported/missing package manager error. For our current use case defaulting to pnpm makes sense and is convenient,  I just don't know if that's an assumption we'll want to make for other projects.

Reply:
Documenting our conversation here. We decided to detect the package manager automatically instead of letting it be passed as a cli flag.